tests/tests/test-stake.ts

```diff
@@ -0,0 +1,36 @@
+import { expect } from "chai";
+import { step } from "mocha-steps";
+
+import { describeWithMoonbeam, createAndFinalizeBlock } from "./util";
+
+describeWithMoonbeam("Moonbeam RPC (Stake)", `simple-specs.json`, (context) => {
+  const GENESIS_ACCOUNT = "0x6be02d1d3665660d22ff9624b7be0551ee1ac91b";
+  const GENESIS_STAKED = "100000";
+  const GENESIS_ACCOUNT_BALANCE = "340282366920938463463374607431768111455";
+  const ACCOUNT_BALANCE1 = "340282366920938463463374607431768121455";
+  const ACCOUNT_BALANCE2 = "340282366920938463463374607431768131455";
```

Comment:
I wouldn't put ACCOUNT_BALANCE2 as a constant here, as it is the result of one of your test
It would be better to have, in your test:
```
const expectedBalance = BigInt(ACCOUNT_BALANCE) + 100_000n;
expect(await context.web3.eth.getBalance(GENESIS_ACCOUNT)).to.equal(expectedBalance.toString());
```